src/main/resources/properties/secure/app-dev.yaml

```diff
@@ -1,10 +1,10 @@
 auth:
   
-  orgId: "![yqKEWZ57M3XUasumfcwhVykLsfUM6yOEv2oEJ04vxZc=]"
+  orgId: "30370ff2-d147-43a4-9402-fb074b715423"
   
   # If auth mode = platform-credentials (ignored, otherwise):
-  username: "![Zjv8nFIUwDlayZ6O7DP31p1zQRfrwfjbkmi5DyNiD38=]"
-  password: "![yqKEWZ57M3XUasumfcwhV35QQl9B5mf5GBon8eY/x8o=]"
+  username: "qamar_"
+  password: "Mulesoftlondon2021!"
```

Comment:
Please remove personal credentials from the commit, in addition to modifying exposed credentials